extension.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "facebook-pixel",
-  "version": "1.1.0",
+  "name": "facebook-pixel-extension",
+  "version": "1.0.1",
   "description": "The Facebook pixel is an analytics tool that allows you to measure the effectiveness of your advertising by understanding the actions people take on your website.",
   "displayName": "Facebook Pixel",
   "iconPath": "resources/icons/facebook.svg",
   "platform": "web",
   "author": {
-    "name": "Adobe Systems",
-    "url": "http://adobe.com",
-    "email": "reactor@adobe.com"
+    "name": "Facebook",
+    "url": "https://developers.facebook.com/support/",
+    "email": "test@test.com"
```

Comment:
I think we can remove the email filed. Tested in sandbox, the initial extension.json doesn't have a required field of email in author. Same as package.json